en/_posts/2016-05-03-how-to-use-deskaas-lab.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title:  "How to subscribe to and use the DeskaaS lab"
+categories: Labs
+author: lrivallain
+lang: en
+---
+
+## Ordering process
+
+Prerequisites: Having a manager access on ovh.com website.
+
+### From RunAbove website
+
+To order : Visit [https://www.runabove.com/deskaas.xml](https://www.runabove.com/deskaas.xml) and clic on "Start Now !"
+
+You'll be redirected to the order wizard and will receive a mail with deskaas login and url.
+
+### From OVH.com manager
+
+Log-in on customer's manager. Deskaas product is available in the "sunrise" part of manager.
+
+Clic on the "**+**" link at the right of "Desktop as a service" ):
+
+![Order](/kb/images/2016-05-03-how-to-use-deskaas-lab/orderfrommanager.png)
+
+You'll be redirected to the order wizard and will receive a mail with deskaas login and url.
+
+## Available offers
+
+Currently, only one offer is available :
+
+*   1 core + 2GB RAM + 10GB Storage + Windows OS
+
+![Lab](/kb/images/2016-05-03-how-to-use-deskaas-lab/offers.png)
+
+It's planned to quickly add more models with more storage, RAM or CPU cores.
+
+
+## Quickstart
+
+### Web client access
+
+The delivery mail contains an URL https://*.deskaas.com to join the web client:
+
+![web_client_welcome](/kb/images/2016-05-03-how-to-use-deskaas-lab/web_client_welcome.png)
+
+Use the mail credentials to log-in:
+
+![web_client_login](/kb/images/2016-05-03-how-to-use-deskaas-lab/web_client_login.png)
+
+Then it's possible to open the virtual desktop:
+
+![web_client_launchitems](/kb/images/2016-05-03-how-to-use-deskaas-lab/web_client_launchitems.png)
+
+Virtual desktop can now be used directly through the web browser:
+
+![web_client_desktop](/kb/images/2016-05-03-how-to-use-deskaas-lab/web_client_desktop.png)
+
+A settings menu is available on the left side of the desk :
+
+![web_client_menu](/kb/images/2016-05-03-how-to-use-deskaas-lab/web_client_menu.png)
+
+
+### Software client
+
+A software client is available on most OS to access to the virtual desktop : [vmware_horizon_clients](https://my.vmware.com/en/web/vmware/info/slug/desktop_end_user_computing/vmware_horizon_clients/4_0)
+
+Once the client is install and launched, you can create a shortcut for the server:
+
+![soft_new-server](/kb/images/2016-05-03-how-to-use-deskaas-lab/soft_new-server.png)
+
+Add the <a>https://*.deskaas.com</a> URI from the mail and validate:
+
+![soft_lists-servers](/kb/images/2016-05-03-how-to-use-deskaas-lab/soft_lists-servers.png)
+
+Click on server to log-in:
+
+![soft_authentication](/kb/images/2016-05-03-how-to-use-deskaas-lab/soft_authentication.png)
+
+Then, the list of available desktop is displayed :
+
+![soft_list-desktops](/kb/images/2016-05-03-how-to-use-deskaas-lab/soft_list-desktops.png)
+
+And virtual desktop is ready to used:
+
+![soft_logged-in](/kb/images/2016-05-03-how-to-use-deskaas-lab/soft_logged-in.png)
```

Comment:
May be interesting to include a useful links section here. Like, where to order, where to get more informations, where is the mailing list, where to get support, ....